### ISSUE TITLE
ddpg.keras-rl.py actual directory 

### ISSUE THREAD

**Issue by fairread** [stanfordnmbl/osim-rl#197]
This could possibly be just a noob mistake for not discovering it easily, but the information in the [Training Your First Models](http://osim-rl.stanford.edu/docs/training/) page could be quite confusing for some. 

In particular, with respect to the latest osim-rl package, the `ddpg.keras-rl.py` script is actually located in the `osim-rl/examples/under-construction` directory rather than the `osim-rl/example` directory. 

Hence the `cd osim-rl/examples` command in the respective page needs to be updated to `cd osim-rl/examples/under-construction` in order to run the following commands.

**Comment by smsong** [comment]
@fairread Yes, you are right. That page is out of date. Please refer to this page: http://osim-rl.stanford.edu/docs/nips2019/training/

**Comment by drozzy** [comment]
But that doesn't work either... @smsong 
Following that http://osim-rl.stanford.edu/docs/nips2019/training/ page, when I execute the:

```
python -m examples.train_arm --train --model sample
```

I get this error:
```
ModuleNotFoundError: No module named 'rl'
```

**Comment by kidzik** [comment]
It just requires `keras-rl`, I'll update the doc, thanks for reporting!
